src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketDefaultBranch.java

```diff
@@ -0,0 +1,92 @@
+package com.atlassian.bitbucket.jenkins.internal.model;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BitbucketDefaultBranch {
```

Comment:
I feel it's a bit more correct for this to inherit from `BitbucketRef`. That will necessitate using the `BitbucketRefType` rather than a String for the type.

Reply:
Oversight on my end. Corrected.